jenkins/spark-nightly-build.sh

```diff
@@ -22,6 +22,9 @@ set -ex
 ## export 'M2DIR' so that shims can get the correct Spark dependency info
 export M2DIR="$WORKSPACE/.m2"
 
+## maven options for building, e.g. '-Dspark-rapids-jni.version=xxx' to specify spark-rapids-jni dependency's version.
+MVN_OPT=${MVN_OPT:-''}
```

Comment:
The build options need to be applied to every mvn cmd

Reply:
One way to help ensure this is to use a variable for the Maven command that includes the desired options, e.g.:
```shell
# use $MVN instead of "mvn" for every invocation of Maven
MVN="mvn ${MVN_OPT}"
```